test/Serilog.Formatting.Compact.Tests/CompactJsonFormatterTests.cs

```diff
@@ -1,98 +1,92 @@
+    }
 
-        [Fact]
-        public void MultiplePropertiesAreDelimited()
-        {
-            AssertValidJson(log => log.Information("Property {First} and {Second}", "One", "Two"));
-        }
+    [Fact]
+    public void MultiplePropertiesAreDelimited()
+    {
+        AssertValidJson(log => log.Information("Property {First} and {Second}", "One", "Two"));
+    }
 
-        [Fact]
-        public void ExceptionsAreFormattedToValidJson()
-        {
-            AssertValidJson(log => log.Information(new DivideByZeroException(), "With exception"));
-        }
+    [Fact]
+    public void ExceptionsAreFormattedToValidJson()
+    {
+        AssertValidJson(log => log.Information(new DivideByZeroException(), "With exception"));
+    }
 
-        [Fact]
-        public void ExceptionAndPropertiesAreValidJson()
-        {
-            AssertValidJson(log => log.Information(new DivideByZeroException(), "With exception and {Property}", 42));
-        }
+    [Fact]
+    public void ExceptionAndPropertiesAreValidJson()
+    {
+        AssertValidJson(log => log.Information(new DivideByZeroException(), "With exception and {Property}", 42));
+    }
 
-        [Fact]
-        public void RenderingsAreValidJson()
-        {
-            AssertValidJson(log => log.Information("One {Rendering:x8}", 42));
-        }
+    [Fact]
+    public void RenderingsAreValidJson()
+    {
+        AssertValidJson(log => log.Information("One {Rendering:x8}", 42));
+    }
 
-        [Fact]
-        public void MultipleRenderingsAreDelimited()
-        {
-            AssertValidJson(log => log.Information("Rendering {First:x8} and {Second:x8}", 1, 2));
-        }
+    [Fact]
+    public void MultipleRenderingsAreDelimited()
+    {
+        AssertValidJson(log => log.Information("Rendering {First:x8} and {Second:x8}", 1, 2));
+    }
 
-        [Fact]
-        public void AtPrefixedPropertyNamesAreEscaped()
-        {
-            // Not possible in message templates, but accepted this way
-            var jobject = AssertValidJson(log => log.ForContext("@Mistake", 42)
-                                                    .Information("Hello"));
+    [Fact]
+    public void AtPrefixedPropertyNamesAreEscaped()
+    {
+        // Not possible in message templates, but accepted this way
+        var jobject = AssertValidJson(log => log.ForContext("@Mistake", 42)
+                                                .Information("Hello"));
 
-            JToken val;
-            Assert.True(jobject.TryGetValue("@@Mistake", out val));
-            Assert.Equal(42, val.ToObject<int>());
-        }
+        Assert.True(jobject.TryGetValue("@@Mistake", out var val));
+        Assert.Equal(42, val.ToObject<int>());
+    }
 
-        [Fact]
-        public void TimestampIsUtc()
-        {
-            // Not possible in message templates, but accepted this way
-            var jobject = AssertValidJson(log => log.Information("Hello"));
+    [Fact]
+    public void TimestampIsUTC()
```

Comment:
`UTC` is an abbreviation, so should be `Utc` (may need to update editorconfig for this?)

Reply:
reverted